pkgs/development/python-modules/psycopg/pool.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, psycopg
+, typing-extensions
+}:
+
+buildPythonPackage {
+  pname = "psycopg-pool";
+  inherit (psycopg) version;
+  format = "setuptools";
+
+  inherit (psycopg) src;
+
+  sourceRoot = "source/psycopg_pool";
+
+  propagatedBuildInputs = lib.optionals (pythonOlder "3.10") [
+    typing-extensions
+  ];
+
+  doCheck = false;
```

Comment:
Because this is tested by `psycopg`'s tests?

Reply:
Yes

Reply:
Might be worth including a comment to this effect.